server/src/modules/genres/genre.entity.ts

```diff
@@ -12,11 +12,11 @@ import { CommonEntity } from 'src/common/entities';
 
 @Entity({ name: 'genres' })
 export class GenreEntity extends CommonEntity {
-  @ApiProperty()
+  @ApiProperty({ type: 'string', format: 'uuid' })
   @PrimaryGeneratedColumn('uuid')
   id: string;
```

Comment:
```suggestion
  readonly id: string;
```

Reply:
I decided to make this changes in another pull request